frontend/src/forms/manage-restaurant-form/ImageSection.tsx

```diff
@@ -9,18 +10,27 @@ import { Input } from '@/components/ui/input';
 import { useFormContext } from 'react-hook-form';
 
 const ImageSection = () => {
-  const { control } = useFormContext();
+  const { control, watch } = useFormContext();
+  const existingImageUrl = watch('imageUrl');
 
   return (
     <div className="space-y-2">
       <div>
         <h2 className="text-2xl font-bold">Image</h2>
         <FormDescription>
-          Add a image that will be displayed on your resataurant listing
+          Add a image that will be displayed on your restaurant listing
         </FormDescription>
       </div>
 
-      <div className="flex flex-col gap-8 w-[50%]">
+      <div className="flex flex-col gap-8 md:w-[50%]">
+        {existingImageUrl && (
+          <AspectRatio ratio={16 / 9}>
```

Comment:
<div>

<strong>Bito Code Review Agent Run</strong> <a href=https://github.com/omkardongre/Food-Ordering-App/pull/33#issuecomment-2394670085>#d60155</a><span> - 10/04/2024, 09:21 pm</span>
<br>

<div><small>🟠</small> Medium importance</div>

<div id="suggestion">
<div id="issue">
<b>Issue:</b> The AspectRatio component is imported but not fully implemented in the ImageSection component.
</div>
<div id="fix">
<b>Fix:</b> Implement the AspectRatio component for the existing image display. Wrap the image element with the AspectRatio component to ensure consistent image proportions.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -26,7 +26,9 @@
          {existingImageUrl && (
            <AspectRatio ratio={16 / 9}>
 +            <img
 +              src={existingImageUrl}
 +              alt="Restaurant image"
 +              className="object-cover w-full h-full rounded-md"
 +            />
            </AspectRatio>
          )}
        </div>
```
</div>
</details>
</div>
<br>


</div>

Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged